### ISSUE TITLE
Could not find any deployments for actor vercel[bot]

### ISSUE THREAD

**Issue by zackcreach** [patrickedqvist/wait-for-vercel-preview#33]
Hello! I am trying to grab my Vercel preview URL to then send it to Cypress for e2e testing.

However, in Github Actions I keep seeing "Could not find any deployments for actor vercel[bot]"

<img width="597" alt="image" src="https://user-images.githubusercontent.com/24227703/164276549-826430d5-d577-42ad-81dd-fb752ab57b11.png">

Here's my config:

```
  test_integration:
    name: Integration tests
    runs-on: ubuntu-latest
    steps:
      - name: Capture Vercel preview URL
        uses: patrickedqvist/wait-for-vercel-preview@v1.2.0
        id: vercel_preview_url
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          environment: 'Preview – website'
          max_timeout: 360

      - uses: actions/checkout@v2
      - name: Run Cypress
        uses: cypress-io/github-action@v1
        with:
          config: baseUrl=${{ steps.vercel_preview_url.outputs.url }}
          install: false
```

Meanwhile Vercel bot is working no problem and deploying successfully:

<img width="546" alt="164276948-535c5aad-981c-446f-8fba-3861f12f3c6f" src="https://user-images.githubusercontent.com/24227703/164316654-c40fca13-c02f-45c8-955c-15a1bc65db7a.png">

Any idea why it's not finding any deployments? I bumped up the `max_timeout` thinking perhaps it was waiting on a completed build but to no avail. 

I feel like I'm missing something simple. Please bail me out if so!

**Comment by patrickedqvist** [comment]
Hello @zackcreach,

My guess is that `Preview – website` is not the correct name for the environment. 

In order to get further, we need to have the answers to these questions:
- if we would list the actual actors of the deployments, what would their names be?
- what are the actual names of the environment?

The information we need can be extracted if you call your own repo and pull a request with this method https://docs.github.com/en/rest/deployments/deployments#list-deployments.

**Comment by zackcreach** [comment]
> Hello @zackcreach,
> 
> My guess is that `Preview – website` is not the correct name for the environment.
> 
> In order to get further, we need to have the answers to these questions:
> 
> * if we would list the actual actors of the deployments, what would their names be?
> * what are the actual names of the environment?
> 
> The information we need can be extracted if you call your own repo and pull a request with this method https://docs.github.com/en/rest/deployments/deployments#list-deployments.

Hey @patrickedqvist thanks for the tip! I was able to pull that list, here's one of the entries: 
```json
  {
    "url": "https://api.github.com/repos/REDACTED/deployments/550777519",
    "id": 550777519,
    "node_id": "DE_kwDOF17qMc4g1DKv",
    "task": "deploy",
    "original_environment": "Preview – website",
    "environment": "Preview – website",
    "description": null,
    "created_at": "2022-04-20T21:30:15Z",
    "updated_at": "2022-04-20T21:30:16Z",
    "statuses_url": "https://api.github.com/repos/REDACTED/deployments/550777519/statuses",
    "repository_url": "https://api.github.com/repos/REDACTED",
    "creator": {
      "login": "vercel[bot]",
      "id": 35613825,
      "node_id": "MDM6Qm90MzU2MTM4MjU=",
      "avatar_url": "https://avatars.githubusercontent.com/in/8329?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/vercel%5Bbot%5D",
      "html_url": "https://github.com/apps/vercel",
      "followers_url": "https://api.github.com/users/vercel%5Bbot%5D/followers",
      "following_url": "https://api.github.com/users/vercel%5Bbot%5D/following{/other_user}",
      "gists_url": "https://api.github.com/users/vercel%5Bbot%5D/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/vercel%5Bbot%5D/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/vercel%5Bbot%5D/subscriptions",
      "organizations_url": "https://api.github.com/users/vercel%5Bbot%5D/orgs",
      "repos_url": "https://api.github.com/users/vercel%5Bbot%5D/repos",
      "events_url": "https://api.github.com/users/vercel%5Bbot%5D/events{/privacy}",
      "received_events_url": "https://api.github.com/users/vercel%5Bbot%5D/received_events",
      "type": "Bot",
      "site_admin": false
    },
    "sha": "c7e79950f389b622445bd075b95fd21fbc8c82c7",
    "ref": "c7e79950f389b622445bd075b95fd21fbc8c82c7",
    "payload": {

    },
    "transient_environment": false,
    "production_environment": false,
    "performed_via_github_app": null
  },
```

It looks like I have the `environment` right (confirmed the en dash too) at least.

After more testing, I've discovered that if I create a brand new PR with this configuration, the plugin works perfectly (successfully finds a preview link)

However, any subsequent pushes to the same branch results in the `Could not find any deployments for actor vercel[bot]` error.

Perhaps this has to do with my `on` config? Here's what I have at the top of my yml file:
```yaml
on:
  pull_request:
    branches:
      - '**'
```

Is this correct?

**Comment by zackcreach** [comment]
Revisited this today. Looks like everything was configured correctly – I just needed to set the timeout longer! Closing issue now.

**Comment by ezeugo-amigo** [comment]
@zackcreach I'm having the same issue. What did you set the timeout to?
